src/main/java/com/exactpro/th2/check1/event/CheckSequenceUtils.java

```diff
@@ -142,6 +145,9 @@ private static String getKeyFields(String name, ValueFilter valueFilter) {
 
     private static String getKeyFields(String name, SimpleFilter valueFilter) {
         if (valueFilter.getKey()) {
+            if (valueFilter.hasSimpleList() && (valueFilter.getOperation() == FilterOperation.IN || valueFilter.getOperation() == FilterOperation.NOT_IN)) {
+                return ", " + name + ' ' + valueFilter.getOperation() + ' ' + valueFilter.getSimpleList().getSimpleValuesList();
```

Comment:
Maybe extract to a separate method and reuse

Reply:
I don't see any benefits from that. In one case we operate with `ValueFilter` object and in another one with `SimpleFilter`. Those objects are different and because of that, the extracted method will have objects we need as parameters. It will make code harder to read in my opinion

Reply:
I meant the format of the result string